homework-10/uniqHalls.js

```diff
@@ -0,0 +1,39 @@
+//import halls array
+
+var halls = require ('./halls.js');
+
+//import Lodash
+
+var _ = require('lodash');
+
+// array of halls with unique names
+
+
+function getFields(input, field) {
+    var output = [];
+    for (var i=0; i < input.length ; ++i)
```

Comment:
Kod for petlje treba pripaziti da length je uvijek za 1 veći od posljednjeg indexa jer indexi počinju s 0.

Iz tog razloga je bolje koristiti map jer ne treba paziti na to da prođemo kroz sve objekte i da ne pređemo dalje

Reply:
Doduše, ovdje sam ja previdio da je < a ne <= kao komparator, tako da je ovaj kod u redu.
Svejedno, dobro je imati na umu ovo gore navedeno.